mkdocs.yml

```diff
@@ -437,6 +437,10 @@ nav:
     - Ampli: data/sdks/ampli-overview.md
     - SDK References:
       - SDK Catalog: data/sdks/sdk-overview.md
+      - Browser 2.0:
```

Comment:
I was thinking we would nest this under `Browser` to limit top level choices. wdyt?

* Browser
** index
** v2
*** index
*** Ampli Codegen
** v1
*** index
*** Ampli Codegen
*** Migration guide (to v2)

Reply:
I think it's confusing if we move the migration guide around since v1/migration was for migration to v1, and should keep the pattern

Reply:
I was thinking Browser/index would redirect to v2/index. Not sure how to handle the "default" page in mkdocs.

Reply:
> I think it's confusing if we move the migration guide around since v1/migration was for migration to v1, and should keep the pattern

I was thinking we could update all SDK migrations to be located in the SDK they are migrating from. Then only provide migration to the latest version. My hope being that this would direct more people to lastest, vs customer using old migrations to other old SDKs. wdyt?

Reply:
I think that works for our intent to promote the latest. However I think for our readers, they'd expect V1 docs next migration to V1, and V2 docs nest to migration to V2, because both pages are instructions for same version of the SDK.

Reply:
This is how it looks like after the change to nav

<img width="287" alt="image" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/20634289/c1a69312-66ac-43b4-a614-7b63e1e51b14">

Reply:
@justin-fiedler how do you feel about this nav? i think i like the Browser and Browser 2.0 separated in top level

Reply:
@justin-fiedler following up! :)